client/src/pages/learn/apis-and-microservices/mongodb-and-mongoose/index.md

```diff
@@ -13,63 +13,68 @@ MongoDB is a database that stores data records (documents) for use by an applica
+### Create a new admin user on the database.
 
-A summary of all your choices should appear, allowing you to change any information provided in the previous steps. Press **Submit Order** to confirm the information.
+- You should be able to see the green **Get Started** button on the bottom left of your screen, you can click this button to see at which step of the process your are in, if you click on it now, you can see the next step is to **Create your first database user**, go ahead and click on that step.
+    - Follow the instructions by clicking on the `Security` tab.
+    - Click on the green **ADD NEW USER** button.
+    - Enter a user name and password and then select **Atlas Admin** under user privileges, remember to store your username and password somewhere safe.
+    - Click on the **ADD USER** green button in the bottom right of the modal.
 
+### Whitelist your IP address
 
-### Create a new admin user on the database
+- If you now click on the green **Get Started** button in the bottom left of your screen, you should see the next step to take highlighted, **Whitelist your IP address**, click on it.
+    - Follow the instructions by clicking on the `IP Whitelist` tab under the `Security` tab.
+    - Click on the green **ADD IP ADDRESS** button.
+    - In the modal, click the **ALLOW ACCESS FROM ANYWHERE** button and you should see `0.0.0.0/0` pre-filled for the whitelist entry field, click the green **Confirm** button.
 
-After you confirmed your configuration, a new sandbox should have been created in the *MongoDB Deployments* section. We are now going to create an administrator, so you can use the database in your application.
+### Connect to your cluster
 
-- Click the newly-created database.
-- Click the *Users* section.
-- Click the **Add database user** button.
-- Input an username and password for your administrator. Do not mark it as read-only or you will not be able to add any information to the database with this user.
+- Clicking on the green **Get Started** button in the bottom left of your screen should now show you the next step, **Connect to your cluster**, click on it.
+    - Follow the instructions by clicking on the `Connect` button in the `Sandbox` section.
+    - In the pop-up modal, click on **Connect Your Application**, a connection string will be displayed, you can copy that connection string by clicking on the `copy` button.
+    - This will be the final URI that you will use to connect to your db, it will look something like this `mongodb+srv://<user>:<password>@<cluster#-dbname>.mongodb.net/test?retryWrites=true`, notice that the `user` and `cluster#-dbname` fields are already filled out for you, all you would need to replace is the `password` field with the one that you created in the previous step. 
 
-### Get the mLab URI
+- That's it! you now have the URI you will add to your application to connect to your database. Keep this URI safe somewhere, so you can use it later!
```

Comment:
```suggestion
- That's it! You now have the URI you will add to your application to connect to your database. Keep this URI safe somewhere, so you can use it later!
```

Reply:
Totally missed that one! Great catch there @ojeytonwilliams.

Reply:
Applied changes.